CONTRIBUTING.md

```diff
@@ -12,7 +12,7 @@ Run `make setup-dev-env` to install local developer tools and run necessary
 services, such as mongodb, for the end-to-end tests.
 
 ## Unit Tests
-Run `make test-unit` to run the unit tests.
+Run `make alldeps` and `make test-unit` to run the unit tests.
```

Comment:
@adyach maybe you could link to the new "Run unit tests" section that you put into `DEVELOPMENT.md`, so you don't have to write this command twice and keep the documentation in-sync. What do you think?

Reply:
very good idea b2d7e75